server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/aws/DockerAwsS3Container.java

```diff
@@ -28,7 +28,7 @@
 
 public class DockerAwsS3Container {
 
-    private static final String AWS_S3_DOCKER_IMAGE = "scality/s3server:mem-6018536a";
+    private static final String AWS_S3_DOCKER_IMAGE = "linagora/cloudserver:8.1.17";
```

Comment:
can you tag it with `mem` please

Reply:
it's up to the configuration/docker configuration while building the image, i'm not sure this is the mem product. but good to have the mem or file indication